CSS/styles.css

```diff
@@ -0,0 +1,286 @@
+body {
+  padding: 0 40px 0 40px;
+  font-family: sans-serif;
+}
+
+#navbar {
```

Comment:
- [ ] Don't use ids to style. Use classes. [Read more here ](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)on reasons why.